libraries/core_libs/consensus/include/dag/block_proposer.hpp

```diff
@@ -21,8 +21,10 @@ class TransactionManager;
 class FullNode;
 class BlockProposer;
 class DagBlock;
+class KeyManager;
 
 using vrf_sk_t = vrf_wrapper::vrf_sk_t;
+using vrf_pk_t = vrf_wrapper::vrf_pk_t;
```

Comment:
having such using in header is bad practice due to namespaces pollution... having it in cpp file is ok but in this case when implementation is in header, full types should be used